pdc/settings.py

```diff
@@ -93,7 +93,7 @@
     'DEFAULT_METADATA_CLASS': 'contrib.bulk_operations.metadata.BulkMetadata',
 
     'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',
-                                'rest_framework.filters.OrderingFilter'),
+                                'pdc.apps.utils.utils.RelatedNestedOrderingFilter'),
```

Comment:
Just a reminder; you need to update the settings_local.py.dist  and make a sync with the DEFAULT_FILTER_BACKENDS value.